target/linux/rockchip/image/rk3399.mk

```diff
@@ -0,0 +1,28 @@
+# 
+# Copyright (C) 2020 Tobias Maedel
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+define Device/pine64_rockpro64-emmc
+	DEVICE_VENDOR := Pine64
+	DEVICE_MODEL := RockPro64
+	DEVICE_VARIANT := emmc
+	DEVICE_DTS := rockchip/rk3399-rockpro64
+	SUPPORTED_DEVICES := pine64,rockpro64
```

Comment:
This will allow crossflashing, but as I understand the PR, this is intended?

Reply:
Yes, this is expected, it's the same device, only with different storage media.

Reply:
I'm probably missing something, but to me it seems like an overkill. What about supporting both eMMC/SD with single bootscript instead?

Reply:
See for example 5f2ff607e286ab92ede986d62b39391557d36c61

Reply:
I'd prefer ynezz solution as well, particularly if this enables us to have one device for one device.

Reply:
Same reason as below. Let's have only one issue for this. I'm closing this one, let's talk in the other one :)

Reply:
It seems like this device is using distroboot, so you should be able to find out from which device was your script loaded via devtype and devnum variables.

Reply:
This is how your boot script is probably called `boot_a_script=load ${devtype} ${devnum}:${distro_bootpart} ${scriptaddr} ${prefix}${script}; source ${scriptaddr}`

Reply:
Oh, thanks. Very good suggestion, that's much neater than my 2 targets. 
It's fixed now, please take another look!

Reply:
Just a nitpick, you should probably rename `mmc.bootscript` to `bootscript` now as it supports mmc/sd.

Reply:
hm, I renamed it from emmc.bootscript to mmc.bootscript because it can now handle both types of mmc devices.